cmd/pr-reminder/main.go

```diff
@@ -458,3 +465,15 @@ func messageUser(user user, slackClient slackClient) error {
 func getLabelMessage(labels []string) string {
 	return fmt.Sprintf(":label: labeled: *%v*", strings.Join(labels[:], ", "))
 }
+
+// hasUnactionableLabels returns whether a PR has any labels which mark a PR
+// that can't be reviewed in its current state
+func hasUnactionableLabels(labels []github.Label) bool {
+	unactionableLabels := getUnactionablePrLabels()
+	for _, label := range labels {
+		if unactionableLabels.Has(label.Name) {
```

Comment:
Check https://pkg.go.dev/k8s.io/apimachinery/pkg/util/sets#String.Intersection

Reply:
the arg `labels` has type `[]github.Label` and unactionableLabels is a set of **strings**, and so I don't see a way how to use the Intersection function without first converting `labels` into a set of strings, which I think would complicate the function even more. but maybe I'm not seeing someting?

Reply:
Potentially you can create a set from the `[]github.labels` too and compare them.

Reply:
You can also check https://pkg.go.dev/k8s.io/test-infra@v0.0.0-20220929080830-4693eb3a938b/prow/github#HasLabels

Reply:
Let's not do another API call for this, that would make this a considerably more expensive tool to run.

Reply:
Ah, you mean this one: https://github.com/kubernetes/test-infra/blob/29364b9d3b47613dbd8db503fe2997fcedb1a30c/prow/github/helpers.go#L45 that is fine to use